### ISSUE TITLE
light problem

### ISSUE THREAD

**Issue by mastergreen15** [bonepl/ChromaLeague#143]
So as far as i know i've dove everything correctly. But whenever i start my game (summoners rift or practice my keyboard lights just freezes while in cycle mode. I have the run window open the entire time and it gives me " Initialized ChromaRestSDK connection" and says that i have joined the game and i have the connection enabled in syntax. I tired the sdk reset command but that also does nothing. Anyone else have this problem I have an ornata v3 x

**Comment by bonepl** [comment]
Have you tried full reinstall od Synapse? With chroma enabled?

**Comment by mastergreen15** [comment]
Yeah i just reinstalled synapse and did a system reboot and still nothing

**Comment by mastergreen15** [comment]
Here's what everything looks like
![gggg](https://user-images.githubusercontent.com/129593984/229292390-9283426a-1e8e-43cd-a25c-18a7f21198cb.png)

**Comment by bonepl** [comment]
From the screenshots and logs it looks fine to me. What do you mean "it freezes while in cycle mode"? The default animation set up in synapse just stop playing when you enter the game, but not Chroma League related animations play (loading screen should play green bar bouncing left - right)?

**Comment by mastergreen15** [comment]
Yes the default light cycle just freezes whenever I start the game.

**Comment by bonepl** [comment]
oh okay, please let me know if it worked for you

**Comment by bonepl** [comment]
> Yes the default light cycle just freezes whenever I start the game.  It's probably a keyboard issue. I'm getting a new one soon so i'll test it out on that.

Hey man let me know if the problem disappeared with a new keyboard.

**Comment by arnowbourdokan** [comment]
``The default animation set up in synapse just stop playing when you enter the game, but not Chroma League related animations play (loading screen should play green bar bouncing left - right)?``    this is exactly what is happening to me. pleas help
![image](https://github.com/bonepl/ChromaLeague/assets/153135598/5187946f-4bec-4ff3-a53b-5842d22f6130)

**Comment by bonepl** [comment]
> ``The default animation set up in synapse just stop playing when you enter the game, but not Chroma League related animations play (loading screen should play green bar bouncing left - right)?``    this is exactly what is happening to me. pleas help
> ![image](https://github.com/bonepl/ChromaLeague/assets/153135598/5187946f-4bec-4ff3-a53b-5842d22f6130)
> 

Hi, this is related to Riot bug mentioned in https://github.com/bonepl/ChromaLeague/issues/146 - it has been reported and we're waiting for their fixes.

**Comment by bonepl** [comment]
hi can you check Chroma League 1.3.0 and let me know if it fixes the issue for you?

**Comment by faberichfuchs** [comment]
Hey, i got the same issue using Chroma League 1.3.0
![explorer_yE7P6V9ORO](https://github.com/bonepl/ChromaLeague/assets/31857634/481b0bfb-4b2c-4a1a-bf98-64546420d0b3)

**Comment by faberichfuchs** [comment]
I have an ornata v3 keyboard, but i just got it very recently, so the keyboard should not be the issue.

Same as with previous users, my lighting just freezes in the current state and chroma league does not change anything during loading screen or during games

**Comment by faberichfuchs** [comment]
First Session was a normal game, second session is a practice tool game just to test if that changed anything

**Comment by bonepl** [comment]
Everything looks perfectly normal on provided screenshots. Like something is hijacking chroma session - are you sure that there is no other integration running in the background that can take over Chroma League session?

Have you tried running Chroma League on a completely fresh, reinstalled Razer Synapse?

**Comment by faberichfuchs** [comment]
I just completely reinstalled synapse and deleted all my profiles etc, now its not changing the "normal" cycle of colors, but its not doing anything either. Also, i tried joining 2 games, the first was detected, the second one was not.
![image](https://github.com/bonepl/ChromaLeague/assets/31857634/bbe97fff-aaf6-47f8-b4a2-d780b9eba795)

**Comment by bonepl** [comment]
Firewall? Or any other antivirus/anti-malware software? Sometimes windows asks if application is safe to run - this also should be allowed (ChromaLeague is not signed by any certificate).

**Comment by faberichfuchs** [comment]
Yes, i just tried reinstalling synpase again, and strangely the only app having issues is chroma league, i also tried opera chroma app and its working fine. I dont know what else to try? I even tried running chroma league with administrative rights but it was unable to find java.exe whe I tried that. 

![image](https://github.com/bonepl/ChromaLeague/assets/31857634/6aff013d-1e55-4d0a-88e2-6c99deae9ca4)

**Comment by faberichfuchs** [comment]
> Firewall? Or any other antivirus/anti-malware software? Sometimes windows asks if application is safe to run - this also should be allowed (ChromaLeague is not signed by any certificate).

Not that I can tell

**Comment by faberichfuchs** [comment]
The moment I enter loading screen, the lights freeze, and the moment i leave the game or to say, the moment chroma league is not supposed to be "taking control" everything is back to normal.

**Comment by bonepl** [comment]
So it looks like ChromaLeague opens and closes Razer Chroma session successfully but any calls to run animation are not played. For now I have no clue what could be causing such behaviour.

**Comment by graagadaan** [comment]
i have the same issue. everything is correct but for me it just doesnt give me any lights at all its fully off. i also have a ornata v3 i just got it new so i would imagine its not the keyboard.

**Comment by bonepl** [comment]
I'm wondering if this issue is related only to Ornata V3. I've read that Razer changed some lightning mode and Ornata V3 doesn't like key based lights which Chroma League uses a lot.

Unfortunately I don't own Ornata V3 to verify that.

**Comment by bonepl** [comment]
hey guys can you check if this release is working for you?
https://mega.nz/file/K5ZmWDxC#sUO9isKzyhsLMYU8oMBH3sV9pWayj1-KCWnKK8-udKo

**Comment by graagadaan** [comment]
it still didnt work it didnt turn off all the lighting in my keyboard but instead paused the setting it was on before the league of legends one should take over.
________________________________
Van: Jakub Andrzejewski ***@***.***>
Verzonden: zondag 14 april 2024 04:56
Aan: bonepl/ChromaLeague ***@***.***>
CC: graagadaan ***@***.***>; Comment ***@***.***>
Onderwerp: Re: [bonepl/ChromaLeague] light problem (Issue #143)


hey guys can you check if this release is working for you?
https://mega.nz/file/K5ZmWDxC#sUO9isKzyhsLMYU8oMBH3sV9pWayj1-KCWnKK8-udKo

**Comment by bonepl** [comment]
Okay, then I'm back at being clueless :/ I thought maybe Ornata V3 requires CHROMA_CUSTOM2 type of events, but it looks like that's not the case. I wonder if there are any Chroma League users with Ornata V3 that have no issues.

**Comment by graagadaan** [comment]
i wonder that aswell. i'd really like to use the keyboard chroma i think its really cool what you've made. if you make any updates or changes make sure to let me know!!
________________________________
Van: Jakub Andrzejewski ***@***.***>
Verzonden: zondag 14 april 2024 11:44
Aan: bonepl/ChromaLeague ***@***.***>
CC: graagadaan ***@***.***>; Comment ***@***.***>
Onderwerp: Re: [bonepl/ChromaLeague] light problem (Issue #143)


Okay, then I'm back at being clueless :/ I thought maybe Ornata V3 requires CHROMA_CUSTOM2 type of events, but it looks like that's not the case. I wonder if there are any Chroma League users with Ornata V3 that have no issues.

**Comment by bonepl** [comment]
hello @faberichfuchs , @graagadaan, @mastergreen15, @arnowbourdokan - I know it has been a while, but can you verify if this is still issue on version pre-2.0?
